kart/point_cloud/import_.py

```diff
@@ -34,11 +34,12 @@
 
 @click.command("point-cloud-import", hidden=True)
 @click.pass_context
+@click.option("--convert-to-copc/--no-convert-to-copc", is_flag=True, default=True)
```

Comment:
not sure why this is an option? I would have thought we'd always want everything in COPC. Why would a user [be able to] disable this?

If it's just for some testing use-case, should add hidden=True to avoid 'supporting' this.

If it's for performance reasons (don't want import to be slow) we might be shooting ourselves in the foot since it'll presumably make data consumption slower later?

Reply:
This spec has been provided by Rob himself. The original plan was to only allow COPC since it's the cooler format that lets us do more stuff - but, newer plan (IIUC) is that sometimes you'll be able to turn a pile of LAZ files that you have in eg S3 into a Kart repo somehow.  I haven't got to that bit of the plan yet - focussing on local only still - but as a concession to it, I'm already allowing repos backed with non-COPC LAZ files (which are commonly used in the real world, but not LAS files, which mostly aren't).